### PR TITLE
ACMS-730: Automated provisioning of Search Cores

### PR DESCRIPTION
**Pull request by chandan-singh7929**
**Motivation**

Fixes #[ACMS-730](https://backlog.acquia.com/browse/ACMS-730)

**Proposed changes**

Update dashboard page for acquia search form to create search core for the environment where the form is being saved 

**Testing steps**

Saving acquia search form should automatically create search core for environment where its being said.
**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: modules/acquia_cms_tour/tests/src/Functional/AcquiaSearchTest.php [hunk 1]
```diff
@@ -70,6 +72,12 @@ public function testAcquiaSearch() {
     // Save Application UUID.
     $dummy_uuid = getenv('SEARCH_UUID');
     $container->fillField('edit-uuid', $dummy_uuid);
+    // Save API key & secret.
+    $dummy_cloud_api_key = getenv('CLOUD_API_KEY');
```
Comment (chandan-singh7929) [inline modules/acquia_cms_tour/tests/src/Functional/AcquiaSearchTest.php:76]: I have added env variable in travis to get this test pass.

File: modules/acquia_cms_tour/tests/src/Functional/AcquiaSearchTest.php [hunk 2]
```diff
@@ -61,6 +61,8 @@ public function testAcquiaSearch() {
     $assert_session->fieldExists('Acquia Subscription identifier');
     $assert_session->fieldExists('Acquia Search API hostname');
     $assert_session->fieldExists('Acquia Application UUID');
+    $assert_session->fieldExists('Acquia API key');
+    $assert_session->fieldExists('Acquia API secret');
```
Comment (chandan-singh7929) [inline modules/acquia_cms_tour/tests/src/Functional/AcquiaSearchTest.php:65]: I have added env variable in travis to get this test pass.

**Comment by vishalkhode1** [comment]
@chandan-singh7929 Please update the PR title/description and branch to ACMS-248.

**Comment by chandan-singh7929** [comment]
Closing this one in favour of #787

**Pull request closed without merging**